packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -64,6 +65,8 @@ import type {
 	TreeObjectNodeUnsafe,
 } from "./typesUnsafe.js";
 import { createFieldSchemaUnsafe } from "./schemaFactoryRecursive.js";
+import { inPrototypeChain, TreeNodeValid } from "./types.js";
+import { UsageError } from "@fluidframework/telemetry-utils/internal";
```

Comment:
We usually put framework imports closer to the top

Reply:
Didn't we have automated import sorting and enforcement at some point? I wonder what happened to that.